app/src/main/java/org/fossasia/openevent/general/sessions/Session.kt

```diff
@@ -1,16 +1,25 @@
 package org.fossasia.openevent.general.sessions
 
+import androidx.annotation.NonNull
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
 import com.fasterxml.jackson.databind.PropertyNamingStrategy
 import com.fasterxml.jackson.databind.annotation.JsonNaming
 import com.github.jasminb.jsonapi.LongIdHandler
 import com.github.jasminb.jsonapi.annotations.Id
 import com.github.jasminb.jsonapi.annotations.Relationship
 import com.github.jasminb.jsonapi.annotations.Type
+import org.fossasia.openevent.general.sessions.microlocation.MicroLocation
+import org.fossasia.openevent.general.sessions.sessiontype.SessionType
 
 @Type("session")
 @JsonNaming(PropertyNamingStrategy.KebabCaseStrategy::class)
+@Entity
```

Comment:
Database schema must have changed? I guess you didn't commit that?

Reply:
I think you mean an autogenerated json file in schema folder. Should I also commit that?

Reply:
I'm uploading it now.